Source/JavaScriptCore/Configurations/Base.xcconfig

```diff
@@ -160,7 +160,7 @@ WK_USE_RESTRICTED_ENTITLEMENTS = $(USE_INTERNAL_SDK);
 // Shared variables used for dynamic or static linking of JavaScriptCore and jsc.
 
 // Tomorrow me will be smart enough to figure out how to do this properly.
-JSC_SEC_LD_FLAGS[sdk=iphoneos*] = -weak_framework ServiceExtensionsCore;
+JSC_SEC_LD_FLAGS[sdk=iphoneos*] = -weak_framework BrowserEngineCore;
```

Comment:
Discussed elsewhere, but I believe we need an additional `JSC_SEC_LD_FLAGS[sdk=iphoneos18.0*] = ;` clause for now.